### ISSUE TITLE
[Bug]: t.assert is not a function in @nolyfill/es-iterator-helpers

### ISSUE THREAD

**Issue by fourdim** [SukkaW/nolyfill#19]
```
TypeError: Error while loading rule 'react/display-name': t.assert is not a function
Occurred while linting /home/project/packages/block-std/src/event/base.ts
    at Object.assert (/home/project/node_modules/.pnpm/@nolyfill+es-iterator-helpers@1.0.20/node_modules/@nolyfill/es-iterator-helpers/dist/CreateIterResultObject.js:1:1885)
    at s (/home/project/node_modules/.pnpm/@nolyfill+es-iterator-helpers@1.0.20/node_modules/@nolyfill/es-iterator-helpers/dist/index4.js:1:4215)
    at MapIterator.t (/home/project/node_modules/.pnpm/@nolyfill+es-iterator-helpers@1.0.20/node_modules/@nolyfill/es-iterator-helpers/dist/implementation2.js:1:851)
    at mergeRules (/home/project/node_modules/.pnpm/eslint-plugin-react@7.33.2_eslint@8.48.0/node_modules/eslint-plugin-react/lib/util/Components.js:273:22)
```

version: `1.0.20`
There is no issue with version `1.0.18`

**Comment by SukkaW** [comment]
@fourdim Thanks for the report! I will look into this.

**Comment by SukkaW** [comment]
@fourdim Fixed in the latest version (`1.0.21`).

**Comment by fourdim** [comment]
Confirmed. No issue now.
